code/index.html

```diff
@@ -2,21 +2,122 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Uplifting News!</title>
+    <link rel="icon" href="images/book.png" />
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Major+Mono+Display&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
   </head>
+  <!-- Icons made by <a href="https://www.flaticon.com/authors/flat-icons" title="Flat Icons">Flat Icons</a> from <a
+  href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a> -->
+
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
+      <div class="left-header-container">
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+
+        <img src="images/book.png" alt="Newspaper logo" />
+      </div>
+
+      <div class="container-upliftingnews">
+        <h1>Uplifting News</h1>
+      </div>
+
+      <nav>
+        <a href="">World</a>
+        <a href="">Local</a>
+        <h1>Uplifting News</h1>
```

Comment:
Try to only use one `<h1>` for each page. The `<h1>` should describe what the page is